app/views/OverviewPageView.scala.html

```diff
@@ -40,7 +40,7 @@ <h2 class="app-task-list__section">
             </li>
             <li class="app-task-list__item">
                 <span class="app-task-list__task-name">
-                    <a class="govuk-link" id="dividends_link" href="@{appConfig.personalIncomeTaxSubmissionDividendsUrl}">
+                    <a class="govuk-link" id="dividends_link" href=@if(incomeSources.dividends==None){"@{appConfig.personalIncomeTaxSubmissionDividendsUrl}"}else{"@{appConfig.personalIncomeTaxSubmissionCYAUrl}"}>
```

Comment:
links not working try doing as screenshot
<img width="870" alt="Screenshot 2020-10-30 at 3 48 31 pm" src="https://user-images.githubusercontent.com/63247344/97726789-865d2f00-1ac7-11eb-920b-0dd0e4200db5.png">